### ISSUE TITLE
EssentialsX & Worldedit

### ISSUE THREAD

**Issue by ghost** [EssentialsX/Essentials#1304]
For some reason my server overrides Worldedits help menu over EssentialsX's.
Do you know anything about it?

Server version: 1.11.2 Spigot
Worldedit: 6.1.7
EssentialsX: 2.0.1

**Comment by SupaHam** [comment]
I don't understand. Can you rephrase that sentence please? What is happening when you type `/help` and what are you expecting it to do?

**Comment by Twonox** [comment]
May be you have set `no-double-slash` to `true` in WorldEdit's config file.
Because `//help` it's WE's help.

**Comment by mdcfe** [comment]
@Twonox I'm guessing you mean in WE's config not in WG's config.

@Ronnie0982 If you want to keep the `no-double-slash` option enabled (assuming you were using it in the first place), I'd suggest using a `commands.yml` alias for `/help` to run `/essentials:help` instead.

**Comment by ghost** [comment]
Yea okay, i have tried some things and after alot of work i found out the my scoreboard plugin was overriding EssentialsX's system..

**Comment by mdcfe** [comment]
If this is still a problem, feel free to reply to this issue and I'll reopen it.
